### ISSUE TITLE
Version

### ISSUE THREAD

**Issue by tensor5** [todevelopers/gnome-shell-extension-rss-feed#6]
Hi,

I installed `rss-feed` from https://extensions.gnome.org/extension/948/rss-feed/ and looking at the local `metadata.json` it shows

```
...
"version": 2
...
```

while [`metadata.json`](https://github.com/todevelopers/gnome-shell-extension-rss-feed/blob/master/rss-feed%40gnome-shell-extension.todevelopers.github.com/metadata.json) shows 1.1, and yet the latest release is 1.2.
Why the version downloaded from gnome website is higher?

Thanks

**Comment by tommy-gun** [comment]
Hi, I made a big mistake here.
Gnome website has own versioning system starting with 1 and increment with 1 automatically at every new extension version is uploaded. 
What confusing me is that current version of extension on gnome portal should be 3... not 2 as you mentioned. 

My mistake is that I try using my own version system at github. First version to 1.0 then 1.1, 1.2 etc. But I forgot write 1.2 to metadata.json. The better way should be use the same system as gnome website.

I try to rework my versions and update metadata.json on github ASAP.

**Comment by tensor5** [comment]
I updated now, and I have version 3. So 3 corresponds to github  version 1.2 right?

**Comment by tommy-gun** [comment]
Yes they are equal. But now I thnik better could be same version number. Because this should be confusing. And maybe some upgrade between github version and gnome portal shouldn't work.

**Comment by tommy-gun** [comment]
Closing as part of a cleanup ahead of the v7 release — the extension has been reworked and ported to GNOME 46+.
